src/nirizan/trust/attribution.py

```
@@ -46,42 +110,60 @@ def analyze(
 
         for name, scores in score_inputs.items():
-            if not scores or not all(np.isfinite(x) for x in scores):
+            try:
+                validate_scores(scores)
+            except ValueError:
                 return AttributionVerdict(
                     attribution=DriftAttribution.INCONCLUSIVE,
                     anchor_set_id=anchor_set_id,
                     system_score_delta=0.0,
                     judge_score_delta=0.0,
                     evaluated_at=datetime.now(timezone.utc),
```

Ruff (0.16.2)

[warning] Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)